backend/vehicles/src/modules/company/company-profiles/profiles/company-profile.profile.ts

```diff
@@ -49,7 +49,7 @@ export class CompanyProfile extends AutomapperProfile {
             }
           }),
         ),
-        forMember((d) => d.companyDirectory, fromValue('bbceid')),
+        forMember((d) => d.companyDirectory, fromValue('BBCEID')),
```

Comment:
Wonder if we should use the standard directory names from WebADE? BCEID being business BCeID, MYID being basic BCeID? If we change this it'll have to be changed throughout.

Reply:
Hi John,

I have sent an email to Chris Robinson asking the same. I will create a new PR upon his confirmation.

Reply:
Sounds good, we can figure out the specifics of the IDs a bit later on.